File: DesktopSample/DataBinding/Section/IListBinding/VB.NET/IListBinding.DataLayer/DataProvider.vb
Imports System.Data.SQLite

Friend Class DataProvider

	'<summary>
	' ProductCollection内のデータの読み込み用の新しい接続オブジェクトを返します。
	'</summary>
	Friend Shared ReadOnly Property NewConnection() As SQLiteConnection
		Get
			Dim connectString As String = My.Resources.ConnectionString
			Return New SQLiteConnection(connectString)
		End Get
	End Property

End Class 'DataProvider

File: DesktopSample/DataBinding/Section/IListBinding/C#/IListBinding.DataLayer/DataProvider.cs
using System.Data.SQLite;

namespace ActiveReports.Samples.IListBinding.DataLayer
{
	internal class DataProvider
	{
		/// <summary>
		/// ProductCollection内のデータの読み込み用の新しい接続オブジェクトを返します。
		/// </summary>
		internal static SQLiteConnection NewConnection
			=> new SQLiteConnection(ActiveReports.Samples.IListBinding.DataLayer.Properties.Resources.ConnectionString);
	}
}
